src/pages/VenuePage/VenuePage.tsx

```diff
@@ -141,7 +145,9 @@ const VenuePage = () => {
   const location = venueName;
   useLocationUpdateEffect(user, venueName);
 
-  const newLocation = { [location]: new Date().getTime() };
+  const newLocation = {
+    [location]: new Date().getTime(),
+  };
```

Comment:
Not sure what the point of this change was. Presumably an explicit comma was added to it, which made `prettier` keep it split across multiple lines.

There's also no need for us to create a `new Date` object just to get the time (see https://github.com/sparkletown/internal-sparkle-issues/issues/20):

```suggestion
  const newLocation = { [location]: Date.now() };
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1016/commits/3a4a3d7ec1e2f6519931827554f4c7b79bf735cf